src/app/components/login/login.component.ts

```diff
@@ -46,8 +45,8 @@ export class LoginComponent implements OnInit, OnDestroy {
     }
 
     if (this.finishLogin) {
-      this.subscription = this.userProfileService.getMe().subscribe((user) => {
-        this.router.navigate(['/home']);
+      this.subscription = this.userProfileService.getMe().subscribe(() => {
+        this.router.navigate(['/']);
       });
       localStorage.getItem('acc');
```

Comment:
what do you use this for?

Reply:
This is Alexandra's code, but I'll try answering the question. What do you mean by 'this'. Are you referring to the subscription variable ? If so, I think she uses that variable in ngOnDestroy to unsubscribe.